uint32      start_time_ns
string[]    macs
RSSI[]      rssi
